src/sunburst.js

```diff
@@ -167,9 +169,9 @@ export default Kapsule({
         (state.onClick || this.focusOnNode)(d.data);
       })
       .on('mouseover', d => {
-        state.tooltip.style('display', 'inline');
+        state.tooltip.style('display', state.tooltipShow(d.data) ? 'inline' : 'none');
         state.tooltip.html(`<div class="tooltip-title">${
-          getNodeStack(d).map(d => nameOf(d.data)).join(' > ')
+          state.tooltipTitle ? state.tooltipTitle(d) : getNodeStack(d).map(d => nameOf(d.data)).join(' > ')
```

Comment:
The method signature should be `state.tooltipTitle(d.data, d)` to be consistent with `tooltipContent`.